Core/src/IfcGeometryConverter/SolidModelConverter.h

```diff
@@ -855,7 +853,8 @@ namespace OpenInfraPlatform
 				throw oip::UnhandledException(sweptAreaSolid);
 			}
 
-			void convertIfcFixedReferenceSweptAreaSolid(EXPRESSReference<typename IfcEntityTypesT::IfcFixedReferenceSweptAreaSolid> fixedRefSweptAreaSolid,
+			void convertIfcFixedReferenceSweptAreaSolid(
+				EXPRESSReference<typename IfcEntityTypesT::IfcFixedReferenceSweptAreaSolid> fixedRefSweptAreaSolid,

```

Comment:
shouldn't this be a `const &` reference?

Reply:
Resolved: 3f70103